autotx/autotx_agent.py

```diff
@@ -17,7 +18,7 @@ def __init__(self) -> None:
             f"{tool.name}: {tool.description}" for tool in self.tools
         ]
 
-    def build_autogen_agent(self, autotx: 'AutoTx', user_proxy: autogen.UserProxyAgent, llm_config: Optional[Dict[str, Any]]) -> autogen.Agent:
+    def build_autogen_agent(self, autotx: 'AutoTx', user_proxy: autogen.UserProxyAgent, llm_config: Optional[Dict[str, Any]], notify_user: Callable[[str, Color | None], None]) -> autogen.Agent:
```

Comment:
nit: instead of passing `notify_user` isnt it possible to use `autotx.notify_user`?